backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/BinaryInterface.kt

```diff
@@ -272,3 +272,9 @@ internal val ClassDescriptor.objectInstanceFieldSymbolName: String
 
 internal val ClassDescriptor.typeInfoHasVtableAttached: Boolean
     get() = !this.isAbstract() && !this.isExternalObjCClass()
+
+internal val String.moduleConstructorName
+    get() = "_Konan_init_${this}"
+
+internal val KonanLibraryReader.moduleConstructorName
+    get() = "_Konan_init_${uniqueName}"
```

Comment:
Maybe better call above method.

Reply:
What would you propose?

Reply:
get() =  uniqueName.moduleConstructorName